library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -1013,18 +1013,14 @@ def Import(devregex, devices)
     # for this hardware
     def GetDeviceTypes
```

Comment:
The GetDeviceTypes is which really show us the options available:

https://github.com/yast/yast-network/blob/90bf250c3cd8becf7f7196dea1e83449adaac900/src/lib/y2network/widgets/interface_type.rb#L52